src/Pages/Users/MyReservationsModal.jsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import classes from './MyReservationsModal.module.css';
+
+const MyReservationsModal = ((props) => {
```

Comment:
I like your solution, but we already installed Bootstrap and that has a model component, so we can use and redesign that the way we want. The creation of duplicate components doesn't seem to be a great idea. What do you think?

Reply:
@aimalamiri I agree, acually I was late to submit my Pr because I was struggling to make bootstrap modal work, I followed all the steps from the documentation but it didn't, that's why I had to make it this way after working on bootstrap for 4 hours.
I have to use bootstrap for my next task, so I'll ask for your help to implement it if you don't mind.